bowtie/templates/report.html.j2

```diff
@@ -170,7 +170,7 @@
 
       <div class="card-body">
         <table class="table table-sm table-hover">
-          <thead class="table-header table-light">
+          <thead>
```

Comment:
Are you sure the `table-header` class also should be removed?

Reply:
Yes, 100% sure, this was used for get this specific table header in JS to implement dark mode for a single table header, But dark mode should be applicable for all tables across whole site. That's why removed this and used dark mode and light mode in whole table not in a single table header